core/src/main/java/org/mapstruct/EnumMapping.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * @author Filip Hrisafov
+ */
+@Target(ElementType.METHOD)
+@Retention(RetentionPolicy.CLASS)
+public @interface EnumMapping {
```

Comment:
What are the reasons to have a new annotation and not reusing `ValueMapping`? Isn't `ValueMapping` also meant for enumerations?

Reply:
I was thinking the same thing. The thing with `@ValueMapping` is that it is closer to `@Mapping`. It has a `source` and `target`. The new `@EnumMapping` is closer to `@BeanMapping`. The idea is to be able to configure general mapping only once for value mappings

Reply:
Ahh, correct! `@ValueMapping` could be used multiple times